Header header
uint8 address
uint8 remote_address
uint32 stamp_us
float32 round_trip_time
float32 range
uint16[3] std_noise
uint16[3] preamble_acc_count
uint16[3] first_path_index
uint16[3] first_path_amp_1
uint16[3] first_path_amp_2
uint16[3] first_path_amp_3
uint16[3] channel_impulse_response_power
uint16[3] prf
